modules/eks-managed-node-group/variables.tf

```diff
@@ -276,7 +276,7 @@ variable "metadata_options" {
 variable "enable_monitoring" {
   description = "Enables/disables detailed monitoring"
   type        = bool
-  default     = true
+  default     = false
```

Comment:
we can't change this since that would be a breaking change, but we can add a note to change the default at the next breaking change. For example https://github.com/terraform-aws-modules/terraform-aws-eks/blob/c63a57908d7b4d51895d3f8502d60daae758b761/variables.tf#L496

Reply:
Great just made the change, thanks!